fetchteiheader.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+import glob
+from datetime import datetime
+from acdh_tei_pyutils.tei import TeiReader, TeiEnricher
+from lxml import etree as ET
+import json
+import os
+import re
+
+source_directory = "tei"
+logn = "fetchteiheader"
+
+with open("mets2tei.json", "r") as f:
+    dictionary = json.load(f)
+
+nsmap = {
+    "tei": "http://www.tei-c.org/ns/1.0",
+    "mets": "http://www.loc.gov/METS/",
+    "mods": "http://www.loc.gov/mods/v3",
+    "dv": "http://dfg-viewer.de/",
+}
+
+
+def log(logname, filename, arg="could not be parsed"):
+    now = datetime.now()
+    with open(f"{logname}.log", "a") as f:
+        f.write(f'{now.strftime("%Y/%m/%d %H:%M:%S")}\t' f"File `{filename}` {arg}\n")
+
+
+def getxml(docid):
```

Comment:
minor thing, but maybe use snake case for better readability, `getxml(docid)` -> `get_xml(doc_id)`